web/discord_relay/queue.go

```diff
@@ -59,8 +54,14 @@ func sendConnectMessage(discord *discordgo.Session, message MessageStruct) {
 		AllowedMentions: &discordgo.MessageAllowedMentions{
 			Parse: []discordgo.AllowedMentionType{},
 		},
-		Content:  fmt.Sprintf("%v %v | %v has connected to the server", JOIN_EMOJI, message.Data.SteamName, message.Data.SteamId),
-		Username: "Relay",
+		Username:  message.Data.SteamName,
+		AvatarURL: message.Data.Avatar,
+		Embeds: []discordgo.MessageEmbed{
+		    &discordgo.MessageEmbed{
+                Description: fmt.Sprintf("%v ***Connected to the server***", JOIN_EMOJI),
+                Color:       65280,
+            },
```

Comment:
This is wrong, I'm not sure how to properly do it; @plally

Reply:
`0x` before the number if its a hex color

Reply:
The embed structure is also wrong, how do I make an array of one embed?

Reply:
```go
[]*discordgo.MessageEmbed{
    {
      Description: "description",
    }
}
```
im not sure you can have an embed without a title can you?